homeassistant/components/xiaomi_miio/vacuum.py

```diff
@@ -326,6 +327,7 @@ def device_state_attributes(self):
                         self.consumable_state.sensor_dirty_left.total_seconds() / 3600
                     ),
                     ATTR_STATUS: str(self.vacuum_state.state),
+                    ATTR_MOP_ATTACHED: bool(self.vacuum_state.is_water_box_attached),
```

Comment:
```suggestion
                    ATTR_MOP_ATTACHED: self.vacuum_state.is_water_box_attached,
```
The property will always return a boolean, so there is no need to cast here.